x-pack/legacy/plugins/fleet/public/pages/agent_details/components/details_section.tsx

```diff
@@ -50,73 +49,74 @@ interface Props {
   onClickUnenroll: () => void;
 }
 export const AgentDetailSection: SFC<Props> = ({ agent, onClickUnenroll, unenrollment }) => {
-  const mapMetadata = (obj: { [key: string]: string } | undefined) => {
-    return Object.keys(obj || {}).map(key => ({
-      key,
-      value: obj ? obj[key] : '',
-    }));
-  };
-
-  const metadataItems = mapMetadata(agent.local_metadata)
-    .concat(mapMetadata(agent.user_provided_metadata))
-    .filter(item => PREFERRED_METADATA.indexOf(item.key) >= 0)
-    .map(item => ({
-      title: getMetadataTitle(item.key),
-      description: item.value,
-    }))
-    .slice(0, MAX_METADATA);
-
+  const metadataFlyout = useFlyout();
   const items = [
+    {
+      title: i18n.translate('xpack.fleet.agentDetails.statusLabel', {
+        defaultMessage: 'Status',
+      }),
+      description: <AgentHealth agent={agent} />,
+    },
     {
       title: i18n.translate('xpack.fleet.agentDetails.idLabel', {
-        defaultMessage: 'Agent ID',
+        defaultMessage: 'ID',
       }),
       description: agent.id,
     },
     {
       title: i18n.translate('xpack.fleet.agentDetails.typeLabel', {
-        defaultMessage: 'Agent Type',
+        defaultMessage: 'Type',
       }),
       description: agent.type,
     },
     {
-      title: i18n.translate('xpack.fleet.agentDetails.lastCheckinLabel', {
-        defaultMessage: 'Last checkin',
+      title: i18n.translate('xpack.fleet.agentDetails.policyLabel', {
+        defaultMessage: 'Policy',
       }),
-      description: agent.last_checkin ? (
-        <FormattedRelative value={new Date(agent.last_checkin)} />
-      ) : (
-        '-'
+      description: (
+        <EuiLink color="text">
+          <FormattedMessage
+            id="xpack.fleet.agentDetails.policyLink"
+            defaultMessage="{policy} (view)"
+            values={{ policy: agent.policy_id }}
+          />
+        </EuiLink>
       ),
     },
-  ].concat(metadataItems);
+  ];
 
   return (
     <>
       <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
         <EuiFlexItem grow={false}>
-          <EuiTitle size="s">
-            <h3>
+          <EuiTitle size="l">
+            <h1>
               <FormattedMessage id="xpack.fleet.agentDetails.title" defaultMessage="Agent Detail" />
-            </h3>
+            </h1>
           </EuiTitle>
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <AgentHealth agent={agent} />
+          <EuiButton
+            disabled={unenrollment.loading === true || agent.active === false}
+            isLoading={unenrollment.loading}
+            onClick={onClickUnenroll}
+          >
+            <FormattedMessage id="xpack.fleet.agentDetails.unenroll" defaultMessage="Unenroll" />
```

Comment:
similar to my other comment, the ID of this one should be something like `xpack.fleet.agentDetails.unenrollButtonLabel` or `...ButtonText`